esmvaltool/cmor/tables/obs4mips/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+If you are *registering content* (RC) for obs4MIPs, please fill out the requested information below.   If you want to create an issue about something else, please delete the text below and title your issue as appropriate.  
```

Comment:
I think it would be a cleaner to just copy the Tables directory into the esmvaltool repository, because that is all we need.

Reply:
I just cloned the repository to the path, as it is done with CMIP5 and CMIP6. I agree that this one is a bit more clutered, so I will leave only the json files

Reply:
Thanks. Did you do this already? Because there is still quite a lot of clutter around.